Failing builds due to merge collision

### Bug description

All builds on main are failing because PR #5013 and PR #5139 were merged - both fine on their own but cause minor conflict together.

```
Error: packages/renderer/src/lib/dashboard/ProviderCard.spec.ts(30,9): error TS2741: Property 'cleanupSupport' is missing in type '{ internalId: string; id: string; extensionId: string; name: string; containerConnections: never[]; kubernetesConnections: never[]; status: "not-installed"; containerProviderConnectionCreation: false; ... 7 more ...; installationSupport: false; }' but required in type 'ProviderInfo'.
Error: packages/renderer/src/lib/dashboard/ProviderCard.spec.ts(55,9): error TS2741: Property 'cleanupSupport' is missing in type '{ internalId: string; id: string; extensionId: string; name: string; containerConnections: never[]; kubernetesConnections: never[]; status: "not-installed"; containerProviderConnectionCreation: false; ... 7 more ...; installationSupport: false; }' but required in type 'ProviderInfo'.
Error: packages/renderer/src/lib/dashboard/ProviderCard.spec.ts(81,9): error TS2741: Property 'cleanupSupport' is missing in type '{ internalId: string; id: string; extensionId: string; name: string; containerConnections: never[]; kubernetesConnections: never[]; status: "not-installed"; containerProviderConnectionCreation: false; ... 7 more ...; installationSupport: false; }' but required in type 'ProviderInfo'.
Error: packages/renderer/src/lib/dashboard/ProviderCard.spec.ts(111,9): error TS2741: Property 'cleanupSupport' is missing in type '{ internalId: string; id: string; extensionId: string; name: string; containerConnections: never[]; kubernetesConnections: never[]; status: "not-installed"; containerProviderConnectionCreation: false; ... 7 more ...; installationSupport: false; }' but required in type 'ProviderInfo'.
Error: packages/renderer/src/lib/dashboard/ProviderCard.spec.ts(137,9): error TS2741: Property 'cleanupSupport' is missing in type '{ internalId: string; id: string; extensionId: string; name: string; containerConnections: never[]; kubernetesConnections: never[]; status: "not-installed"; containerProviderConnectionCreation: false; ... 8 more ...; version: string; }' but required in type 'ProviderInfo'.
Error: packages/renderer/src/lib/dashboard/ProviderCard.spec.ts(164,9): error TS2741: Property 'cleanupSupport' is missing in type '{ internalId: string; id: string; extensionId: string; name: string; containerConnections: never[]; kubernetesConnections: never[]; status: "not-installed"; containerProviderConnectionCreation: false; ... 7 more ...; installationSupport: false; }' but required in type 'ProviderInfo'.
```

### Operating system

macOS 14.2

### Installation Method

None

### Version

next (development version)

### Steps to reproduce

See any PR built since this morning.

### Relevant log output

_No response_

### Additional context

_No response_

--- a/packages/renderer/src/lib/dashboard/ProviderCard.spec.ts
+++ b/packages/renderer/src/lib/dashboard/ProviderCard.spec.ts
@@ -44,6 +44,7 @@ test('Expect provider region', async () => {
     warnings: [],
     images: {} as ProviderImages,
     installationSupport: false,
+    cleanupSupport: false,
   };
   render(ProviderCard, { provider });
 
@@ -69,6 +70,7 @@ test('Expect provider name', async () => {
     warnings: [],
     images: {} as ProviderImages,
     installationSupport: false,
+    cleanupSupport: false,
   };
   render(ProviderCard, { provider });
 
@@ -95,6 +97,7 @@ test('Expect provider icon', async () => {
     warnings: [],
     images: { icon: 'test.png' } as ProviderImages,
     installationSupport: false,
+    cleanupSupport: false,
   };
 
   render(ProviderCard, { provider });
@@ -125,6 +128,7 @@ test('Expect no provider version', async () => {
     warnings: [],
     images: {} as ProviderImages,
     installationSupport: false,
+    cleanupSupport: false,
     // no version
   };
   render(ProviderCard, { provider });
@@ -152,6 +156,7 @@ test('Expect provider version', async () => {
     images: {} as ProviderImages,
     installationSupport: false,
     version: '1.2.3',
+    cleanupSupport: false,
   };
   render(ProviderCard, { provider });
 
@@ -178,6 +183,7 @@ test('Expect provider state', async () => {
     warnings: [],
     images: {} as ProviderImages,
     installationSupport: false,
+    cleanupSupport: false,
   };
   render(ProviderCard, { provider });
 

